### PR TITLE
Project creation

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/test/resources/features/login.feature [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Feature: Login The Ninja Store
+  Scenario: Login The Ninja Store to an existing account
+    Given the user enters his email and password
+    When the user presses login button
+    Then user login successfully
```
Comment (titor-oopart) [inline src/test/resources/features/login.feature]: add empty line

File: src/main/java/theNinjaStore/ui/pages/CalculadoraPage.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
+package theNinjaStore.ui.pages;
```
Comment (titor-oopart) [inline src/main/java/theNinjaStore/ui/pages/CalculadoraPage.java]: delete this class

File: src/main/java/theNinjaStore/ui/utils/Operations.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/*
```
Comment (titor-oopart) [inline src/main/java/theNinjaStore/ui/utils/Operations.java]: delete operation class

File: src/main/java/ninjaStore/ui/pages/PageTransporter.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+package ninjaStore.ui.pages;
+
```
Comment (titor-oopart) [inline src/main/java/ninjaStore/ui/pages/PageTransporter.java:13]: add author

File: src/test/java/hooks/Hooks.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+package hooks;
+
+import cucumber.api.java.After;
+
```
Comment (titor-oopart) [inline src/test/java/hooks/Hooks.java:15]: add author

File: src/test/java/runner/RunCukesTest.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+package runner;
+
```
Comment (titor-oopart) [inline src/test/java/runner/RunCukesTest.java:13]: add author

File: src/test/resources/features/login.feature [hunk 7]
```diff
@@ -0,0 +1,5 @@
+Feature: Login The Ninja Store
+  Scenario: Login The Ninja Store to an existing account
+    Given the user enters his email and password
+    When the user presses login button
+    Then successful login
```
Comment (titor-oopart) [inline src/test/resources/features/login.feature]: add empty line

File: src/test/resources/features/login.feature (hunk 1 above)
Comment (MelissaRoman06) [reply src/test/resources/features/login.feature]: done

File: src/main/java/theNinjaStore/ui/pages/CalculadoraPage.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/theNinjaStore/ui/pages/CalculadoraPage.java]: done

File: src/main/java/theNinjaStore/ui/utils/Operations.java (hunk 3 above)
Comment (MelissaRoman06) [reply src/main/java/theNinjaStore/ui/utils/Operations.java]: done

File: src/main/java/ninjaStore/ui/pages/PageTransporter.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/main/java/ninjaStore/ui/pages/PageTransporter.java:13]: done

File: src/test/java/hooks/Hooks.java (hunk 5 above)
Comment (MelissaRoman06) [reply src/test/java/hooks/Hooks.java:15]: done

File: src/test/java/runner/RunCukesTest.java (hunk 6 above)
Comment (MelissaRoman06) [reply src/test/java/runner/RunCukesTest.java:13]: done

File: src/test/resources/features/login.feature (hunk 7 above)
Comment (MelissaRoman06) [reply src/test/resources/features/login.feature]: done

File: src/main/java/core/selenium/webdrivers/Chrome.java [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+import java.util.HashMap;
+
+/**
+ * Chrome class sets the browser.
+ *
+ * @author Melissa Román
+ * @version 1.0
+ */
+public class Chrome implements IBrowser {
+    /**
```
Comment (johnpiterzon) [inline src/main/java/core/selenium/webdrivers/Chrome.java]: add line

File: src/main/java/ninjaStore/ui/pages/LoginPage.java [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * LoginPage class models the used WebElements and actions for that page.
+ *
+ * @author Melissa Román
+ * @version 1.0
+ */
+public class LoginPage extends BasePage {
+    /**
```
Comment (johnpiterzon) [inline src/main/java/ninjaStore/ui/pages/LoginPage.java]: add empty line

Comment (MelissaRoman06) [reply src/main/java/ninjaStore/ui/pages/LoginPage.java]: done

**Pull request merged**